doc/sphinx/proof-engine/ltac.rst

```diff
@@ -1705,13 +2209,17 @@ performance issue.
 
    This flag enables and disables the profiler.
 
-.. cmd:: Show Ltac Profile
+.. cmd:: Show Ltac Profile {? {| CutOff @int | @string } }
```

Comment:
@JasonGross Why can't the user provide both a `string` and a `CutOff`?

Reply:
It'd be pretty easy to implement that, just change https://github.com/coq/coq/blob/0ae20176df146b90f11e131aff3673584e6ffb3a/plugins/ltac/profile_ltac.ml#L452-L454
to pass along a `cutoff` argument, change https://github.com/coq/coq/blob/0ae20176df146b90f11e131aff3673584e6ffb3a/plugins/ltac/profile_ltac.mli#L53 to match, and then update the command syntax to allow it at https://github.com/coq/coq/blob/0ae20176df146b90f11e131aff3673584e6ffb3a/plugins/ltac/profile_ltac_tactics.mlg#L75-L82

Reply:
As for why it's not already implemented, I think it was never deemed especially useful.  In fact the only cutoff I've ever used other than the default one is `0`, to see everything, and usually I don't want to see everything but only under a specific string.

Reply:
@Zimmi48 It might be helpful if GitHub had a checkmark emoji that reviewers could add to indicate they're done with a conversation without resolving it (maybe with the convention that the PR submitter resolves things.  Or if 2 reviewers have commented in a single item, would make it easy to see if both have read the last comment.  WDYT?  👍 ?

Reply:
Feel free to send feedback to GitHub: https://support.github.com/contact/feedback
Don't hesitate to link to the PR to show the kind of size that this can get.